x-pack/legacy/plugins/lens/public/xy_visualization/xy_expression.tsx

```diff
@@ -26,17 +27,26 @@ import {
 import { EuiIcon, EuiText, IconType, EuiSpacer } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { i18n } from '@kbn/i18n';
-import { FormatFactory } from '../legacy_imports';
+import { EmbeddableVisTriggerContext } from '../../../../../../src/plugins/embeddable/public';
+import { VIS_EVENT_TO_TRIGGER } from '../../../../../../src/legacy/core_plugins/visualizations/public/np_ready/public/embeddable/events';
+import { FormatFactory } from '../../../../../../src/legacy/ui/public/visualize/loader/pipeline_helpers/utilities';
 import { LensMultiTable } from '../types';
 import { XYArgs, SeriesType, visualizationTypes } from './types';
 import { VisualizationContainer } from '../visualization_container';
 import { isHorizontalChart } from './state_helpers';
+import { UiActionsStart } from '../../../../../../src/plugins/ui_actions/public';
 
 type InferPropType<T> = T extends React.FunctionComponent<infer P> ? P : T;
 type SeriesSpec = InferPropType<typeof LineSeries> &
   InferPropType<typeof BarSeries> &
   InferPropType<typeof AreaSeries>;
 
+export interface XYChartSeriesIdentifier {
+  yAccessor: string | number;
+  splitAccessors: Map<string | number, string | number>; // does the map have a size vs making it optional
+  seriesKeys: Array<string | number>;
+}
```

Comment:
It seems this type is now exposed by the chart library itself and we don't need to put this here manually.